src/app/backend/dashboard.go

```diff
@@ -29,9 +30,16 @@ var (
 		"to connect to in the format of protocol://address:port, e.g., "+
 		"http://localhost:8080. If not specified, the assumption is that the binary runs inside a"+
 		"Kubernetes cluster and local discovery is attempted.")
+	argHeapsterHost = pflag.String("heapster-host", "", "The address of the Heapster Apiserver "+
+		"to connect to in the format of protocol://address:port, e.g., "+
+		"http://localhost:8082. If not specified, the assumption is that the binary runs inside a"+
+		"Kubernetes cluster and local discovery is attempted.")
 )
 
 func main() {
+        // When Heapster host not defined then heapster client will be null
```

Comment:
It is only for the time being? Later there will be in cluster client? If yes, please add a todo.